.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -29,7 +29,7 @@
   split it into multiple PRs. This makes things easier and faster to code review.
 -->
 
-- [ ] Dependency upgrade
+- [ ] Dependency upgrade (please include link to diff between library versions if possible)
```

Comment:
We normally ask for a link to the changr.log and as fall back a link to a diff.

Reply:
We either require one or more of the following:

- A link to the release notes of this package version, and all versions in between.
- A link to the changelog of this package.
- A link to a Git(Hub) diff/compare view from the current version to the bumped version.

IMHO It is hard to incorporate all that in a checkbox item...

Reply:
I think this text should go under the `Checklist` section and not under the `Type of change` section.